cmd/crio/main.go

```diff
@@ -484,6 +484,7 @@ func main() {
 
 		go func() {
 			service.StartExitMonitor()
+			service.StartHooksMonitor()
```

Comment:
Took me some time to figure out that the previous line is blocking.
You need to have `service.StartHooksMonitor` in another goroutine

Reply:
oh gosh, thanks I was struggling to figure that out as well

Reply:
fixed